lib/utils/indent-ts.js

```diff
@@ -1302,6 +1325,42 @@ function defineVisitor({
         setOffset(atToken, 0, tokenStore.getFirstToken(decorators[0]))
       }
     },
+    AccessorProperty(node) {
+      const keyNode = node.key
+      const valueNode = node.value
+      const firstToken = tokenStore.getFirstToken(node)
+      const keyTokens = getFirstAndLastTokens(keyNode)
+      const prefixTokens = tokenStore.getTokensBetween(
+        firstToken,
+        keyTokens.firstToken
+      )
+      if (node.computed) {
+        prefixTokens.pop() // pop [
```

Comment:
Leftover comment?

```suggestion
        prefixTokens.pop()
```

Reply:
That pop, pops the `[` token, so it's not a leftover.

Reply:
Ah, so maybe better

```suggestion
        prefixTokens.pop() // pop opening bracket character (`[`)
```

Reply:
Thanks!